core/src/main/java/com/googlecode/psiprobe/beans/stats/providers/MultipleSeriesProvider.java

```diff
@@ -65,9 +60,9 @@ public int getMovingAvgFrame() {
 
     /**
      * @param movingAvgFrame - if this value is greater than 0, a moving avg value is calculated for every series using
-     * every Nth value, where N % movingAvgFrame == 0. Top series are identified based on a max moving avg
-     * value of each series. If the movingAvgFrame equals to 0, top series are determined based on a simple avg
-     * of all series values.
+     *                       every Nth value, where N % movingAvgFrame == 0. Top series are identified based on a max moving avg

```

Comment:
More horizontal alignment, this time in Javadoc. Style guide will probably disallow this.